src/ui.bar.js

```diff
@@ -100,57 +92,9 @@ let _ = Mavo.UI.Bar = class Bar {
 			}
 		}
 
-		if (this.controls.length && !this.noResize) {
-			this.resize();
-
-			if (self.ResizeObserver) {
-				this.resizeObserver = Mavo.observeResize(this.element, entries => {
-					this.resize();
-				});
-			}
-		}
-
 		Mavo.observers.resume();
```

Comment:
Do we still need it here?

Reply:
I have no idea. Let's leave it for now, can't hurt.